sn_aia_agent_080b5e79378422100d1bdfc734924b3a.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update sys_domain="global" table="sn_aia_agent">
```

Comment:
The following issues were found in the file:
- **1**: The `record_type` tag must be set to `template`. The current value is `external`.
- **2**: The `sys_created_by` tag value must be `admin`. The current value is `abhishek.suryvanshi`.